win32ss/gdi/ntgdi/freetype.c

```diff
@@ -6795,7 +6795,8 @@ IntExtTextOutW(
     FONT_CACHE_ENTRY Cache;
     FT_Matrix mat;
     BOOL bNoTransform;
-    DWORD ch0, ch1, etx = 3; // etx is ASCII End of Text
+    DWORD ch0, ch1, nbsp = 0xa0; // nbsp is a non-breaking space
+    DWORD del = 0x7f; // DEL is ASCII DELETE
```

Comment:
Nit, it might make sense to make `del` and `nbsp` const?

Reply:
or as in Serge's PR, using #defines. (`#define ASCII_ETX 0x03`, `#define ASCII_DEL 0x7f`, etc.)

Reply:
I changed them to ```const``` as Mark suggested. @HBelusca, using ````#define```'s would have been good as well. Thanks.